package.json

```diff
@@ -11,7 +11,7 @@
   "version": "2.0.0-alpha",
   "license": "MIT",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=7.0.0"
```

Comment:
Please merge `2` into your branch again. We’re setting this to 6.9.1 for the `2` branch. The tests will need to be able to run in both 6.9.1 and 7+ harmony.